administrator/components/com_admin/script.php

```diff
@@ -229,6 +230,7 @@ protected function uninstallExtensions()
              * 'pre_function' => Name of an optional migration function to be called before
              *                   uninstalling, `null` if not used.
              */
+            ['type' => 'plugin', 'element' => 'updatenotification', 'folder' => 'system', 'client_id' => 0, 'pre_function' => 'updatenotification_plugin_migration'],
```

Comment:
```suggestion
            ['type' => 'plugin', 'element' => 'updatenotification', 'folder' => 'system', 'client_id' => 0, 'pre_function' => 'migrateUpdatenotificationPlugin'],
```
We're normally using camel case and not underscores for function names.